jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/BaseEnricher.java

```diff
@@ -102,7 +102,7 @@ protected Configuration getConfiguration() {
         return enricherContext.getConfiguration();
     }
 
-    protected String getConfig(Configs.Config key) {
+    public String getConfig(Configs.Config key) {
```

Comment:
why is the change of access needed?

Reply:
I did it to access VolumePermissionEnricher's Config fields from PersistentVolumeClaimStorageClassEnricher .

Reply:
OK, that reveals a big code-smell.
In any case (now that I'm seeing the affected part) if you wanted to expose that method you could have done that at the VolumePermissionEnricher level (by overriding the method and making it public).

Reply:
So,for now, the recommended changes would be to remove this change of access here, and change it at the VolumePermissionEnricher level with a `// TODO` comment linking an issue where we indicate that we need to remove that once the deprecated configs are finally removed in a next version.